golang/cosmos/proto/agoric/swingset/swingset.proto

```diff
@@ -0,0 +1,34 @@
+syntax = "proto3";
+package agoric.swingset;
+
+import "gogoproto/gogo.proto";
+import "cosmos/base/v1beta1/coin.proto";
+
+option go_package = "github.com/Agoric/agoric-sdk/golang/cosmos/x/swingset/types";
+
+message Params {
+    option (gogoproto.equal) = true;
+    option (gogoproto.goproto_stringer) = false;
+
+    // The value of a given named unit in SwingSet "beans".
```

Comment:
The vocabulary of "named units" should either be defined here, or a reference to the definition should be given here.

Reply:
I tried harder with this, hopefully, it looks better now.